extras/signers.json

```diff
@@ -14,7 +14,8 @@
     "Mike": "0x91f4Df13350083eDb30515019cF527C6b0490c54",
     "Nico": "0x815d654E930E840D0E0Ee1B18FFc8Fb4ddA4c6B3",
     "Fernando": "0xbbF0Ae5195444264364CA7eb7E3BB1971B4c3eCb",
-    "EmergencyPauser": "0x170027069fd114BFF2f57B0FC796df93290C02a6"
+    "EmergencyPauser": "0x170027069fd114BFF2f57B0FC796df93290C02a6",
+    "Fabio": "0x90347b9CC81a4a28aAc74E8B134040d5ce2eaB6D"
```

Comment:
@mendesfabio could you confirm you control this address/is for balancer work?  Also if you have time to check the rest of the addresses and approve the PR great.